build.sh

```diff
@@ -1,14 +1,13 @@
 #!/bin/bash
-
 dotnet restore
-
-cd src/Serilog/
-
-dotnet build -f netstandard1.0 -c Release
-dotnet build -f netstandard1.3 -c Release
-
-cd ../..
-cd test/Serilog.Tests/ 
-
-dotnet build -f netcoreapp1.0 -c Release
-dotnet test -f netcoreapp1.0  -c Release
\ No newline at end of file
+for path in src/*/project.json; do
+    dirname="$(dirname "${path}")"	
+    dotnet build ${dirname} -f netstandard1.0 -c Release
+    dotnet build ${dirname} -f netstandard1.3 -c Release
```

Comment:
One more stupid question; why are we hard coding these? If you don't pass anything, won't it build all supported targets? I'm afraid that if we change the targets in the future, we'll forget to update these and things will break. It's not a big deal, but it would be nice if it just built everything.